.eslintrc.cjs

```diff
@@ -63,6 +87,10 @@ module.exports = {
 				"selector": 'ImportDeclaration > Literal[value=/\\bsrc\\b/]',
 				"message": "Import from the `lib` dir instead of the `src` dir."
 			},
+			{
+				"selector": 'CallExpression[callee.name=/describe|it/] > ObjectExpression > Property[key.name="only"][value.raw="true"]',
+				"message": "Use { only: true } for local testing only. Remember to remove it before merging to main."
+			},
```

Comment:
The node test runner doesn't have something equivalent to mocha's `--forbid-only` argument for running on CI/CD so these new rules have been added in ESLint to catch them here instead.